### ISSUE TITLE
Feedback from Alex 

### ISSUE THREAD

**Issue by akleefel** [TedHaley/Sommelier#1]
Hey Ted,

I really liked your app. I was impressed by your conceptual thoughts behind the layout, the realistic usage scenario as well as the very informative value chart in the second tab.

Here are my suggestions for improvements:

* Only after talking to you, I realized the user should pick wine attributes based on their preferred flavors in the first tab - and then, based on their selection, review the wine options in the second tab. It would be great to have a more visible and descriptive disclaimer at the top of the landing page. Something like: "Welcome to my wine selection tool. Please, use the filters to select wines and review your options on the second tab".

* When using the app, I was confused about the meaning of the words presented in the word cloud. I believe that a word cloud is a reasonable way to present the information that you want to present (most prevalent flavors in the selected sample of wines). But again, it would be nice to have a heading to give the user some context on what is presents.

* The regression and residual-lines in your value map (second tab) were slightly confusing and do not add a lot of value. Since you are simply trying to present the wine options and their "value" (price vs. rating) I would suggest removing the lines and having a scatter plot with colors according to the "value".
